pandas/io/clipboards.py

```diff
@@ -83,7 +85,9 @@ def read_clipboard(sep=r"\s+", **kwargs):  # pragma: no cover
     return read_csv(StringIO(text), sep=sep, **kwargs)
 
 
-def to_clipboard(obj, excel=True, sep=None, **kwargs):  # pragma: no cover
+def to_clipboard(
+    obj, excel: bool | None = True, sep: str | None = None, **kwargs
+) -> None:  # pragma: no cover
```

Comment:
is `# pragma: no cover` for codecov? Does it need to be on the `def to_clipboard`-line?